experimental/fluent/agentic/src/main/java/io/serverlessworkflow/fluent/agentic/dsl/AgenticDSL.java

```diff
@@ -140,6 +140,10 @@ public static AgentTaskConfigurer agent(Object agent) {
     return list -> list.agent(agent);
   }
 
+  public static AgentTaskConfigurer conditional(Predicate<?> predicate, Object agent) {
+    return list -> list.agent(agent).when(predicate);
+  }
```

Comment:
Add a new overload `conditional(predicate, agents..)` in case we can call a sequence of agents under a certain condition.